org-roam-db.el

```diff
@@ -306,6 +313,17 @@ connections, nil is returned."
     files))
 
 ;;;;; Updating
+(defun org-roam-db--update-meta ()
+  "Update the metadata of the current buffer into the cache."
+  (let* ((file (file-truename (buffer-file-name)))
+         (attr (file-attributes file))
+         (atime (file-attribute-access-time attr))
+         (mtime (file-attribute-modification-time attr)))
+    (org-roam-db-query [:delete :from files
+                        :where (= file $s1)]
+                       file)
+    (org-roam-db--insert-meta file (list :atime atime :mtime 0))))
```

Comment:
```suggestion
    (org-roam-db--insert-meta file (list :atime atime :mtime mtime))))
```

Reply:
Fixed in b33ec3ee522603e1659a0aa499609d582d046bda.